openedx/core/djangoapps/user_authn/api/form_fields.py

```diff
@@ -37,7 +84,7 @@ def _add_field_with_configurable_select_options(field_name, field_label, error_m
     return field_attributes
 
 
-def add_level_of_education_field():
+def add_level_of_education_field(is_field_required):
```

Comment:
In my opinion, it would be a better approach to pass `is_field_required` with a default value so that any new implementation around form fields would not break any logic.